src/external/interfaces/IOwnable.sol

```diff
@@ -0,0 +1,22 @@
+/**
```

Comment:
Without having looking at this too extensively, is this because OZ doesn't offer the `IOwnable`?

Reply:
I'm sorry, I don't understand x) What is the purpose of this file? It doesn't seem to be used anywhere.